server/server.go

```diff
@@ -3,20 +3,36 @@ package server
 import (
 	"archive/tar"
 	"archive/zip"
+	"bytes"
 	"coderunner/constants"
 	"encoding/json"
 	"fmt"
 	"io"
 	"log"
 	"mime/multipart"
 	"net/http"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"strings"
 	"sync"
 )
 
+type assignmentData struct {
+	CommandToExecute string
+	CommandToCompile string
+	WorkDir          string
+	RootDir          string
+	Output           string
+	CmdlineArgs      map[string]string
+}
+
+var AssignmentData assignmentData
```

Comment:
this variable naming is confusing.
Why is the variable public while the struct is package private?

Reply:
Changed it to private